shared/src/test/groovy/gov/hhs/cdc/trustedintermediary/external/hapi/HapiFhirResourceTest.groovy

```diff
@@ -0,0 +1,26 @@
+package gov.hhs.cdc.trustedintermediary.external.hapi
+
+import gov.hhs.cdc.trustedintermediary.context.TestApplicationContext
+import gov.hhs.cdc.trustedintermediary.wrappers.Logger
+import org.hl7.fhir.r4.model.Bundle
+import spock.lang.Specification
+
+class HapiFhirResourceTest extends Specification {
```

Comment:
I guess this test file should be deleted since this is testing test code.

Reply:
Good catch. I will delete this file once the pr is approved.